frontend/javascripts/oxalis/view/action-bar/download_modal_view.tsx

```diff
@@ -670,23 +672,27 @@ function _DownloadModalView({
           >
             Mag
           </Divider>
-          <Row>
-            <Col span={19}>
-              <MagSlider magnificationInfo={selectedLayerMagInfo} value={mag} onChange={setMag} />
-            </Col>
-            <Col
-              span={5}
-              style={{ display: "flex", justifyContent: "flex-end", alignItems: "center" }}
-            >
-              {mag.join("-")}
-            </Col>
-          </Row>
+          {!onlyOneMagAvailable && (
+            <Row>
+              <Col span={19}>
+                <MagSlider magnificationInfo={selectedLayerMagInfo} value={mag} onChange={setMag} />
+              </Col>
+              <Col
+                span={5}
+                style={{ display: "flex", justifyContent: "flex-end", alignItems: "center" }}
+              >
+                {mag.join("-")}
+              </Col>
+            </Row>
+          )}
           <Text
             style={{
               margin: "0 6px 12px",
               display: "block",
             }}
           >
+            {onlyOneMagAvailable && mag.join("-")}
+            <br />
```

Comment:
This added `<br>` also adds a newline after the slides which leads to additional vertical spacing which is not needed imo.
E.g. using 
```suggestion
            {onlyOneMagAvailable && <div>{mag.join("-")}</div>}

```
would avoid this.

Reply:
nice, thanks for catching this!